applications/crossbar/src/modules/cb_storage.erl

```diff
@@ -452,15 +452,17 @@ validate_attachment_settings_fold(AttId, Att, ContextAcc) ->
     Random = kz_binary:rand_hex(16),
     Content = <<"some random content: ", Random/binary>>,
     AName = <<Random/binary, "_test_credentials_file.txt">>,
-    DbName = cb_context:account_db(ContextAcc),
+    DbName = kz_util:format_account_db(cb_context:account_id(ContextAcc)),
     DocId = doc_id(ContextAcc),
```

Comment:
you need to create a new document (storage_settings_probe) and put it in modb then use that document to add the attachments to